.github/workflows/tests.yml

```
@@ -17,15 +17,15 @@ jobs:
       MPLBACKEND: Agg
       CYTHON_TRACE: 1
       CYTHONSPEC: cython
-      NUMPY_MIN: numpy==1.22.4
+      NUMPY_MIN: numpy==1.23.0
       CYTHON_MIN: cython==0.29.35
```

```suggestion
      CYTHON_MIN: cython==3.0.4
```

Should this be kept in sync with pyproject.toml